edit-post/assets/stylesheets/_mixins.scss

```diff
@@ -309,11 +309,14 @@
 	content: "";
 	pointer-events: none;
 	display: block;
-	position: absolute;
 	width: 0;
 	height: 0;
 	border-left: 3px solid transparent;
 	border-right: 3px solid transparent;
 	border-top: 5px solid currentColor;
-	right: 6px;
+	margin-left: $grid-size-small;
+
+	// This gives the icon space on the right side consistent with the material
```

Comment:
@tofumatt are there any standards documented around ideal line length for comments like this?

Reply:
No, sorry! Basically the rule is most of us set our visible tab length to two/four-width so I just made it work according to that size. I dunno, it probably was me being overly picky; I mostly just wanted to add the full-stop 😅

Reply:
@tofumatt No no it's totally fine! Truthfully I'm never sure and honestly go back and forth on this myself. I think it's worth trying to keep it standardised where possible. I work in a 120 col terminal though so this line length is totally fine.

Reply:
Indeed! I actually think we should have an editorconfig that makes explicit the tab width. The WordPress coding standards are quite deviant compared to most other JS projects, so we should make it easier for others to see how we work.